code/modules/pda/mob_hunt_game_app.dm

```diff
@@ -0,0 +1,182 @@
+
+/*
+	This stuff isn't included with in core_apps.dm because it is so distinct and would end up making the file huge (more than it already is).
+	I put it in it's own file in order to help it stay organized and easier to locate the procs and such.
+	For reference, the other Nano-mob Hunter GO files are in /code/game/modules/arcade/mob_hunt
+*/
+
+/datum/data/pda/app/mob_hunter_game
+	name = "Nano-Mob Hunter GO"
+	icon = "gamepad"
+	template = "pda_mob_hunt"
+	category = "Games"
+
+	var/list/my_collection = list()
+	var/current_index = 0
+	var/connected = 0
+	var/hacked = 0		//if set, this cartridge is able to spawn trap mobs from its collection (set via emag_act on the cartridge)
+	var/catch_mod = 0	//used to adjust the likelihood of a mob running from this client, a negative value means it is less likely to run (support for temporary bonuses)
+	var/wild_captures = 0		//used to track the total number of mobs captured from the wild (does not count card mobs) by this client
+	var/scan_range = 3			//maximum distance (in tiles) from which the client can reveal nearby mobs
+
+/datum/data/pda/app/mob_hunter_game/start()
+	..()
+	processing_objects.Add(pda)
```

Comment:
It would seem that it requires this to be activated/turned on for the PDA to be added to the processing list, then.